app/views/pregnancies/_call_log.html.erb

```diff
@@ -0,0 +1,16 @@
+<section id="call_log" class="call-info tab-pane active">
+	<div id="urgent_pregnancies" class="margin-bottom">
+		<h1 class="border-bottom title"> Call Log</h1>
+		<table class="table border-side">
+		<tr>
+			<th>Date</th>
+			<th>Time</th>
+			<th>Result</th>
+			<th>user</th>
+		</tr>
+		<tbody id="call-log-content" >
+			<%= render 'call_log_content', calls: @pregnancy.calls.order("created_at DESC").take(10) %>
```

Comment:
What do you think about changing this `take` to a `limit`? I think if we're only using the first ten calls on load, no point in grabbing all of them.